analysis_driver/transfer_data.py

```diff
@@ -11,105 +9,29 @@
 
 
 def prepare_run_data(dataset):
-    """
-    Decide whether to rsync a dataset to an intermediate dir and run driver.pipeline on it.
-    :param Dataset dataset: A dataset object
-    """
     app_logger.debug('Preparing dataset %s (%s)', dataset.name, dataset.dataset_status)
-    if cfg.get('intermediate_dir'):
-        _transfer_run_to_int_dir(
-            dataset,
-            cfg['input_dir'],
-            cfg['intermediate_dir'],
-            repeat_delay=int(cfg.get('tt_agent_delay', 120))
-        )
-        dataset_dir = cfg['intermediate_dir']
-    else:
-        dataset_dir = cfg['input_dir']
-
-    return os.path.join(dataset_dir, dataset.name)
+    return os.path.join(cfg['input_dir'], dataset.name)
 
 
 def prepare_sample_data(dataset):
-    """
-    Decide whether to rsync the fastq files to an intermediate dir just find them.
-    :param Dataset dataset: A dataset object
-    """
     app_logger.debug('Preparing dataset %s (%s)', dataset.name, dataset.dataset_status)
     fastqs = []
-
     for run_element in dataset.run_elements:
         if int(run_element.get(ELEMENT_NB_READS_CLEANED, 0)) > 0:
             fastqs.extend(_find_fastqs_for_sample(dataset.name, run_element))
     return fastqs
 
 
 def _find_fastqs_for_sample(sample_id, run_element):
-    run_id = run_element.get(ELEMENT_RUN_NAME)
-    project_id = run_element.get(ELEMENT_PROJECT_ID)
-    lane = run_element.get(ELEMENT_LANE)
-
-    local_fastq_dir = os.path.join(cfg['input_dir'], run_id)
+    local_fastq_dir = os.path.join(cfg['input_dir'], run_element.get(ELEMENT_RUN_NAME))
     app_logger.debug('Searching for fastqs in ' + local_fastq_dir)
-    fastqs = util.find_fastqs(local_fastq_dir, project_id, sample_id, lane)
-    if fastqs:
-        return fastqs
-
```

Comment:
Similarly here, remote data transfer for finding fastqs has been removed as we no longer use it. `prepare_sample_data` may also become vestigial.